conf/test.config

```diff
@@ -29,4 +29,5 @@ params {
     busco_reference               = "https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2020-03-06.tar.gz"
     busco_clean                   = true
     gtdb                          = false
+    cat_official_taxonomy         = true
```

Comment:
As far as I know CAT isnt run in the test profile anyway, because the db is just too large for CI tests.
So I propose removing this line.
But CAT is run in https://github.com/nf-core/mag/blob/master/conf/test_full.config, if you want you could activate cat_official_taxonomy there?

Reply:
Ah, that's a good shout - I'll make that change!